requirements/readme.md

```diff
@@ -1 +1,5 @@
-All development and normal use requirements are in requirements.txt - we use a specific version of gunicorn to run the g2p studio in production, and don't want to introduce git as a dependency, so we separate these requirements here.
\ No newline at end of file
+All requirements for normal use are in `requirements.txt`.
+
+We use a specific version of gunicorn to run the g2p studio in production, and don't want to introduce git as a dependency, so we separate these requirements here into `requirements.prod.txt`.
+
+Requirements recommended for use during development but not needed for running g2p are separated into `requirements.dev.txt`.
```

Comment:
I don't think we ever black-ified `g2p` - what did we decide about this? that we were just going to do it piece meal? Ultimately there are a lot of contributors so it might be hard to enforce this.

Reply:
We had decided to go ahead and blackify it all, but did not follow through yet. I still think we should, though maybe not demand it of external contributors. But it's still useful to have "recommended" filters.